cachito/web/models.py

```diff
@@ -199,8 +199,12 @@ def to_purl(self):
             suffix = match.group("suffix")
             has_authority = match.group("has_authority")
             if protocol == "file":
```

Comment:
Should this be nested under the `not has_authority` branch? Or do we want to accept `file://`?

Reply:
AFAIK, just `file:` is supported [1]. I will move it there.

[1] https://docs.npmjs.com/files/package.json#local-paths

Reply:
Done. Also rebased on master